pymc3/sampling.py

```diff
@@ -561,15 +587,27 @@ def sample(
         f'took {trace.report.t_sampling:.0f} seconds.'
     )
 
+    idata = None
+    if compute_convergence_checks or return_inferencedata:
+        if 'data.save_log_likelihood' in arviz.rcParams:
+            # if the user change the arviz default, use it
+            save_ll = arviz.rcparams['data.save_log_likelihood']
+        else:
+            # otherwise, use the arviz default, which is True as of v0.8.0
+            save_ll = True
+        idata = arviz.from_pymc3(trace, log_likelihood=save_ll, model=model)
```

Comment:
how about using some `idata_kwargs` or something like this? This way users can change pass dims and coords to `from_pymc3` and even a `prior` dict result of `pm.sample_prior_predictive`

Reply:
Good idea! I did this in the latest commit.